.travis/run_job.py

```diff
@@ -86,30 +77,40 @@ def should_run_sbt_project(repo, project_name, changed_paths):
 
     travis_event_type = os.environ["TRAVIS_EVENT_TYPE"]
     travis_build_stage = os.environ["TRAVIS_BUILD_STAGE_NAME"]
+    travis_commit_range = os.environ["TRAVIS_COMMIT_RANGE"]
 
     try:
         # If it's not an sbt task, we always run it no matter what.
         task = os.environ["TASK"]
     except KeyError:
-        sbt_project_name = get_project_name()
+        parser = argparse.ArgumentParser()
+        parser.add_argument("project_name", default=os.environ.get("SBT_PROJECT"))
+        parser.add_argument("--changes-in", nargs="*")
+        args = parser.parse_args()
 
-        repo = Repository(".sbt_metadata")
+        task = f"{args.project_name}-test"
 
+        change_globs = args.changes_in + [".travis.yml"] if args.changes_in else None
```

Comment:
nit: maybe pull this on to a separate line? I don't find one-line ternaries especially used in Python.